modP2pImpl/src/org/aion/p2p/impl1/tasks/TaskReceive.java

```diff
@@ -41,6 +41,7 @@ public void run() {
                     try {
                         hlr.receive(mi.getNodeId(), mi.getDisplayId(), mi.getMsg());
                     } catch (Exception e) {
+                        e.printStackTrace();
```

Comment:
Why print the stack trace here?

Reply:
oops, leftover from when i was prototyping